openedx/core/djangoapps/programs/utils.py

```diff
@@ -3,13 +3,16 @@
 import datetime
 import logging
 from urlparse import urljoin
+from pytz import utc
+
+from xmodule.fields import Date
```

Comment:
This shouldn't end up staying here, but if it does, this is a first-party package and should be in the third and final block of imports.